src/ioc/Ioc.js

```diff
@@ -0,0 +1,17 @@
+export class Ioc {
+    constructor() {
+        this.services = {};
+    }
+
+    register(service, serviceName) {
+        if (this.services[serviceName]) {
+            console.log("Service already created.")
```

Comment:
Throw an error here instead

Reply:
Also you won't need a `if / else` block if you do the above. I think just an if will `work`